### PR TITLE
Add Simplified Chinese Translation

### PR DESCRIPTION
**Pull request by CJYKK**
### Description
Add a new language translation file.

---
**Target Minecraft Versions:** any

### CODE REVIEW

**Review by DelayedGaming — CHANGES_REQUESTED** [review]
Oops, you left Line 59 unchanged.
```
script disabled: <gold>%s<reset> 已禁用！使用 <gray>/<gold>skript <cyan>enable <red>%s<reset> 来启用。
```
My suggestion for Line 153:
```
hooks:
	hooked: 成功与%s连接 。
	error: 无法与%1$s连接。这可能是因为Skript不支持已安装的%1$s版本。
```
Other than mentioned above, translations are accurate!

**Comment by CJYKK** [comment]
> Oops, you left Line 59 unchanged.
> 
> ```
> script disabled: <gold>%s<reset> 已禁用！使用 <gray>/<gold>skript <cyan>enable <red>%s<reset> 来启用。
> ```
> 
> My suggestion for Line 153:
> 
> ```
> hooks:
> 	hooked: 成功与%s连接 。
> 	error: 无法与%1$s连接。这可能是因为Skript不支持已安装的%1$s版本。
> ```
> 
> Other than mentioned above, translations are accurate!

Fixed :)

**Review by DelayedGaming — APPROVED** [review]

File: src/main/resources/config.sk [hunk 1]
```diff
@@ -31,7 +31,7 @@
 # ==== General Options ====
 
 language: english
-# Which language to use. Currently English, German, Korean, French and Polish are included in the download, but custom languages can be created as well.
+# Which language to use. Currently English, German, Korean, French, Polish and Simplified Chinese are included in the download, but custom languages can be created as well.
```
Comment (TheLimeGlass) [inline src/main/resources/config.sk:34]:
```suggestion
# Which language to use. Currently English, German, Korean, French, Polish and Simplified Chinese (Chinese as the name) are included in the download, but custom languages can be created as well.
```
When you set the language it follows <language>.lang so adding these braces allows the user to know exactly what to type as the node.

**Review by TheLimeGlass — CHANGES_REQUESTED** [review]

File: src/main/resources/config.sk (hunk 1 above)
Comment (CJYKK) [reply src/main/resources/config.sk]: I changed the "chinese.lang" file name to "simplifiedchinese.lang", and added some comments in config.sk.
I think using spaces in "language" might cause some problems.

**Review by TheLimeGlass — APPROVED** [review]

**Comment by AllenSheep** [comment]
I am very looking forward to adding Chinese translation for skript, which will make it easier for me to use

**Review by APickledWalrus — APPROVED** [review]

**Pull request merged**